boxsdk/util/translator.py

```diff
@@ -191,5 +193,28 @@ def translate(self, session, response_object):
 
         return translated_obj
 
+    def untranslate(self, value):
+        """
+        Untranslates a given object into a dictionary.
+
+        :param value:
+            The object to untranslate.
+        :rtype value:
+            `dict`
+        :return:
+            A dictionary containing the untranslated object.
+        """
+        if hasattr(value, '_response_object') and isinstance(value._response_object, dict):  # pylint:disable=protected-access
```

Comment:
Another reason to put this on `BaseAPIJSONObject`: because you're accessing one of its private fields.

Rather than do `hasattr(value, '_response_object')`, it would be stronger to do `isinstance(value, BaseAPIJSONObject)`